avocado/core/job.py

```diff
@@ -251,32 +251,33 @@ def _make_test_runner(self):
         self.test_runner = test_runner_class(job=self,
                                              test_result=self.result_proxy)
 
-    def _set_output_plugins(self):
-        if getattr(self.args, 'test_result_classes', None) is not None:
-            for klass in self.args.test_result_classes:
-                test_result_instance = klass(self)
-                self.result_proxy.add_output_plugin(test_result_instance)
-        else:
-            self.result_proxy.add_output_plugin(result.Result(self))
-
-    def _make_test_result(self):
+    def _make_old_style_test_result(self):
         """
-        Set up output plugins.
+        Old style result output plugins setup.
 
-        The basic idea behind the output plugins is:
+        This supports the activation of old style result classes which are
+        registered with :func:`avocado.core.result.register_test_result_class`.
```

Comment:
nitpick: I'd say it should be `avocado.core.result.registered_test_result_class` but it's intended for removal and suits the purpose well. So ack to this :+1: